packages/babel-plugin-redux-saga/README.md

```diff
@@ -6,47 +6,26 @@ Example of setup and demo are available [here](../../examples/error-demo)
 
 ## Example
 
-Source:
-
-```js
-// src/sagas/index.js
-function* saga1(){
-    yield call(foo, 1, 2, 3);
-}
-
-function* saga2(){
-    yield 2;
-}
+Error message without plugin
+```
+The above error occurred in task throwAnErrorSaga
+    created by errorInCallAsyncSaga
+    created by takeEvery(ACTION_ERROR_IN_CALL_ASYNC, errorInCallAsyncSaga)
+    created by rootSaga
 ```
 
-Result:
+Error message with the plugin
+```
+The above error occurred in task throwAnErrorSaga  src/sagas/index.js?16
+    created by errorInCallAsyncSaga  src/sagas/index.js?25
+    created by takeEvery(ACTION_ERROR_IN_CALL_ASYNC, errorInCallAsyncSaga)
+    created by rootSaga  src/sagas/index.js?78
+```
 
-```js
-function* saga1() {
-    yield Object.defineProperty(call(foo, 1, 2, 3), Symbol.for("@@redux-saga/LOCATION"), {
-        value: {
-            fileName: "src/sagas/index.js",
-            lineNumber: 1,
-            code: "call(foo, 1, 2, 3)"
-        }
-    })
-}
+## Install
 
-Object.defineProperty(saga1, Symbol.for("@@redux-saga/LOCATION"), {
-  value: {
-    fileName: "src/sagas/index.js",
-    lineNumber: 1
-  }
-})
-function* saga2() {
-    yield 2;
-}
-Object.defineProperty(saga2, Symbol.for("@@redux-saga/LOCATION"), {
-  value: {
-    fileName: "src/sagas/index.js",
-    lineNumber: 5
-  }
-})
+```sh
+npm i -D babel-plugin-redux-saga
```

Comment:
-D is a short hand that many people might not recognize, let's just use more explicit `--save-dev` for clarity

Reply:
Done!